packages/dds/quorum/src/interfaces.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ISharedObject, ISharedObjectEvents } from "@fluidframework/shared-object-base";
+
+export interface IQuorumEvents extends ISharedObjectEvents {
+    /**
+     * Notifies when a new value goes pending or has been accepted.
+     */
+    (event: "pending" | "accepted", listener: (key: string) => void);
```

Comment:
i like having `, target: IEventThisPlaceHolder` as the last param in events. i makes it easier to write reusable code as the handler doesn't need to know what how to get the quorum object, as it's always given